maintainers/maintainer-list.nix

```diff
@@ -1989,6 +1989,10 @@
     github = "Hodapp87";
     name = "Chris Hodapp";
   };
+  hochi = {
```

Comment:
Please include this in a separate commit

Reply:
So you want me to amend this and force push this branch? Or is there another way I did not now about yet? :)

Reply:
Yeah just rewrite the history into 2 commits and force push please.